Polyfills/Canvas/Source/Image.h

```diff
@@ -41,7 +41,8 @@ namespace Babylon::Polyfills::Internal
         std::string m_src{};
 
         JsRuntimeScheduler m_runtimeScheduler;
-        Napi::FunctionReference m_onloadHandlerRefs;
+        Napi::FunctionReference m_onloadHandlerRef;
+        std::shared_ptr<arcana::cancellation_source> m_cancellationSource{};
```

Comment:
Who calls cancel on this? Maybe dispose should?

Reply:
Indeed!